x/mock/simulation/types.go

```diff
@@ -70,16 +70,3 @@ type WeightedOperation struct {
 	Weight int
 	Op     Operation
 }
-
-// TODO remove? not being called anywhere
-// PeriodicInvariant returns an Invariant function closure that asserts a given
-// invariant if the mock application's last block modulo the given period is
-// congruent to the given offset.
-func PeriodicInvariant(invariant Invariant, period int, offset int) Invariant {
```

Comment:
Why is this removed? I think we should be using it in gaia for the slower invariants, it is quite useful.

Reply:
This code isn't being called anywhere therefor it should be removed - BUT I just opened up an issue with this code in it so it doesn't get lost when we choose to introduce PeriodicInvariants! Thanks for mentioning this

https://github.com/cosmos/cosmos-sdk/issues/2731